base/deprecated.jl

```diff
@@ -1758,4 +1758,7 @@ end)
 @deprecate(SharedArray{T}(filename::AbstractString, ::Type{T}, dims::NTuple, offset; kwargs...),
            SharedArray{T,length(dims)}(filename, dims, offset; kwargs...))
 
+# LibGit2.owner -> LibGit2.repository for consistency
+eval(Base.LibGit2, :(Base.@deprecate_binding owner repository))
```

Comment:
The problem is that `@deprecate` (and presumably `@deprecate_binding`) both export. This came up in #20104. Perhaps we need a non-exporting `@deprecate`?